### DISCUSSION TITLE
Parallax scrolling

### DISCUSSION THREAD

**Discussion by BrookeOlivia** [w3c/wcag#2553] [Q&A]
My company has a client who is potentially wanting a page to use parallax scrolling.  There will be links and content users  could click on in each section, videos, and maybe some other interactive pieces.  They only want to be WCAG AA compliant, but i am finding trouble identifying what specific WCAG items would fall into the parallax scrolling realm.  So far I have the following list:

* Prefers-reduced motion and a manual switch to turn on/off (would be on by default)
* No auto playing videos (> 5 seconds),
* Reflow and zoom need to be supported,
* Contrasts for all texts + interactive elements,
* alternative text possibilities for images that are not decorative,
* video captions for videos that are not decorative,
* keyboard navigation (should scroll with arrows/tabs - aka some keyboard trigger that will need to be told to the user as well)
* Focus on elements

What am I missing and can any provide more specific WCAG guidance that I can use?  Also are there any examples of fully accessible parallax scrolling on a site?  I would love all insight on pros and cons from an accessibility standpoint.

**Comment by Ryladog** [comment]
2.2.2 Stop, Pause, Hide
